components/Opportunities/index.jsx

```diff
@@ -0,0 +1,105 @@
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import {
+  Button, Card, Col, Row, Typography,
+} from 'antd';
+import styled from 'styled-components';
+import opportunities from './opportunities.json';
+
+const LAUNCH_CONTACT_URL = 'https://t.me/pahlmeyer';
+
+const StyledCard = styled(Card)`
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  .ant-card-body {
+    min-height: 100%;
+    display: flex;
+  }
+  margin-bottom: 24px;
+`;
+
+const OpportunityCard = ({
+  agentName,
+  agentDescription,
+  project,
+  image,
+  background,
+}) => (
+  <StyledCard>
+    <Row style={{ columnGap: '12px', width: '100%', minHeight: '100%' }}>
+      <Col xs={24} md={14}>
+        <h2>{agentName}</h2>
+        <p style={{ whiteSpace: 'pre-line' }}>
+          {agentDescription}
+        </p>
+      </Col>
+      <Col xs={24} md={8} style={{ fontSize: '0.875rem', lineHeight: '1.25rem' }}>
+        <div style={{ display: 'flex', flexDirection: 'row', gap: '16px' }}>
```

Comment:
can be Flex component from antd, flexDirection is row anyway, so you can just specify gap={16}
https://ant.design/components/flex/